x-pack/plugins/xpack_main/public/services/path.js

```diff
@@ -9,8 +9,8 @@ import chrome from 'ui/chrome';
 export function PathProvider($window) {
   const path = chrome.removeBasePath($window.location.pathname);
   return {
-    isLoginOrLogout() {
-      return path === '/login' || path === '/logout';
+    isLoginOrLogoutOrLoggedOut() {
```

Comment:
nit: the naming here feels like a bit much to me. Would something like `isUnauthenticatedPath` be appropriate, or is that conveying the wrong thing?

Reply:
Much better name!